percy/snapshot.py

```diff
@@ -18,6 +19,14 @@
 
 # for logging
 LABEL = '[\u001b[35m' + ('percy:python' if PERCY_DEBUG else 'percy') + '\u001b[39m]'
+CDP_SUPPORT_SELENIUM = (str(SELENIUM_VERSION)[0].isdigit() and int(
+    str(SELENIUM_VERSION)[0]) >= 4) if SELENIUM_VERSION else False
+fetched_widths = {}
+
+def log(message, lvl = 'info'):
+    requests.post(f'{PERCY_CLI_API}/percy/log',
```

Comment:
its not necessary that user is also updating cli [ like in js sdks ] so this needs to happen async and need to suppress exceptions

Reply:
Wrap over `try/except` and have `print` in `finally`